manifests/base/server/argocd-server-deployment.yaml

```diff
@@ -251,6 +251,12 @@ spec:
                 name: argocd-cmd-params-cm
                 key: server.k8sclient.retry.base.backoff
                 optional: true
+        - name: ARGOCD_API_CONTENT_TYPES
+          valueFrom:
+            configMapKeyRef:
+              name: argocd-cmd-params-cm
+              key: server.api.content.types
+              optional: true
```

Comment:
not: spacing seems to be off

Reply:
Fixed! The new block was okay but a bunch of others were indented by 4, which is weird.